app/models/item.rb

```diff
@@ -19,6 +19,22 @@ def self.disabled_only
     where(enabled: false)
   end
 
+  def self.top_5_by_revenue 
+    joins(:invoice_items)
```

Comment:
@mkrumholz Could you do `.joins(:invoice_items {invoices: :transactions})`?

Reply:
Ah yea, it looks like that works with some minor adjustments -> `.joins(invoice_items: {invoice: :transactions})`

Reply:
Nice!